[zydis] Build error on x64-mingw-dynamic

Package: zydis:x64-mingw-dynamic@4.1.0

**Host Environment**

- Host: x64-windows
- Compiler: Clang 19.1.7
-    vcpkg-tool version: 2025-02-11-bec4296bf5289dc9ce83b4f5095943e44162f9c2
    vcpkg-scripts version: 5f8c424 2025-02-21 (2 days ago)

**To Reproduce**

`vcpkg install zydis --triplet=x64-mingw-dynamic --host-triplet=x64-mingw-dynamic`

**Failure logs**

```
Downloading https://github.com/zyantific/zydis/archive/v4.1.0.tar.gz -> zyantific-zydis-v4.1.0.tar.gz
Successfully downloaded zyantific-zydis-v4.1.0.tar.gz
-- Extracting source C:/msys64/mnt/vcpkg/downloads/zyantific-zydis-v4.1.0.tar.gz
-- Applying patch zycore.patch
-- Using source at C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean
-- Found external ninja('1.12.1').
-- Configuring x64-mingw-dynamic
-- Building x64-mingw-dynamic-dbg
CMake Error at scripts/cmake/vcpkg_execute_build_process.cmake:134 (message):
    Command failed: C:/msys64/clang64/bin/cmake.exe --build . --config Debug --target install -- -v -j33
    Working Directory: C:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg
    See logs for more information:
      C:\msys64\mnt\vcpkg\buildtrees\zydis\install-x64-mingw-dynamic-dbg-out.log

Call Stack (most recent call first):
  installed/x64-mingw-dynamic/share/vcpkg-cmake/vcpkg_cmake_build.cmake:74 (vcpkg_execute_build_process)
  installed/x64-mingw-dynamic/share/vcpkg-cmake/vcpkg_cmake_install.cmake:16 (vcpkg_cmake_build)
  ports/zydis/portfile.cmake:27 (vcpkg_cmake_install)
  scripts/ports.cmake:196 (include)
```

<details><summary>C:\msys64\mnt\vcpkg\buildtrees\zydis\install-x64-mingw-dynamic-dbg-out.log</summary>

```
Change Dir: 'C:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg'

Run Build Command(s): C:/msys64/clang64/bin/ninja.exe -v -v -j33 install
[1/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/MetaInfo.c.obj -MF CMakeFiles\Zydis.dir\src\MetaInfo.c.obj.d -o CMakeFiles/Zydis.dir/src/MetaInfo.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/MetaInfo.c
[2/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Register.c.obj -MF CMakeFiles\Zydis.dir\src\Register.c.obj.d -o CMakeFiles/Zydis.dir/src/Register.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Register.c
[3/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Zydis.c.obj -MF CMakeFiles\Zydis.dir\src\Zydis.c.obj.d -o CMakeFiles/Zydis.dir/src/Zydis.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Zydis.c
[4/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Segment.c.obj -MF CMakeFiles\Zydis.dir\src\Segment.c.obj.d -o CMakeFiles/Zydis.dir/src/Segment.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Segment.c
[5/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Mnemonic.c.obj -MF CMakeFiles\Zydis.dir\src\Mnemonic.c.obj.d -o CMakeFiles/Zydis.dir/src/Mnemonic.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Mnemonic.c
[6/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Utils.c.obj -MF CMakeFiles\Zydis.dir\src\Utils.c.obj.d -o CMakeFiles/Zydis.dir/src/Utils.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Utils.c
[7/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/String.c.obj -MF CMakeFiles\Zydis.dir\src\String.c.obj.d -o CMakeFiles/Zydis.dir/src/String.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/String.c
[8/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/FormatterBuffer.c.obj -MF CMakeFiles\Zydis.dir\src\FormatterBuffer.c.obj.d -o CMakeFiles/Zydis.dir/src/FormatterBuffer.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/FormatterBuffer.c
[9/21] C:\msys64\clang64\bin\windres.exe -O coff -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -I C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -I C:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -I C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -I C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include  C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/resources/VersionInfo.rc CMakeFiles/Zydis.dir/resources/VersionInfo.rc.obj
FAILED: CMakeFiles/Zydis.dir/resources/VersionInfo.rc.obj
C:\msys64\clang64\bin\windres.exe -O coff -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -I C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -I C:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -I C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -I C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include  C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/resources/VersionInfo.rc CMakeFiles/Zydis.dir/resources/VersionInfo.rc.obj
fatal error: UTF-16 (LE) byte order mark detected in 'C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/resources/VersionInfo.rc', but encoding is not supported
1 error generated.
llvm-rc: Preprocessing failed.
[10/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Disassembler.c.obj -MF CMakeFiles\Zydis.dir\src\Disassembler.c.obj.d -o CMakeFiles/Zydis.dir/src/Disassembler.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Disassembler.c
[11/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Formatter.c.obj -MF CMakeFiles\Zydis.dir\src\Formatter.c.obj.d -o CMakeFiles/Zydis.dir/src/Formatter.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Formatter.c
[12/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/FormatterATT.c.obj -MF CMakeFiles\Zydis.dir\src\FormatterATT.c.obj.d -o CMakeFiles/Zydis.dir/src/FormatterATT.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/FormatterATT.c
[13/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/FormatterIntel.c.obj -MF CMakeFiles\Zydis.dir\src\FormatterIntel.c.obj.d -o CMakeFiles/Zydis.dir/src/FormatterIntel.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/FormatterIntel.c
[14/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/FormatterBase.c.obj -MF CMakeFiles\Zydis.dir\src\FormatterBase.c.obj.d -o CMakeFiles/Zydis.dir/src/FormatterBase.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/FormatterBase.c
[15/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Decoder.c.obj -MF CMakeFiles\Zydis.dir\src\Decoder.c.obj.d -o CMakeFiles/Zydis.dir/src/Decoder.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Decoder.c
[16/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/Encoder.c.obj -MF CMakeFiles\Zydis.dir\src\Encoder.c.obj.d -o CMakeFiles/Zydis.dir/src/Encoder.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/Encoder.c
[17/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/DecoderData.c.obj -MF CMakeFiles\Zydis.dir\src\DecoderData.c.obj.d -o CMakeFiles/Zydis.dir/src/DecoderData.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/DecoderData.c
[18/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/EncoderData.c.obj -MF CMakeFiles\Zydis.dir\src\EncoderData.c.obj.d -o CMakeFiles/Zydis.dir/src/EncoderData.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/EncoderData.c
[19/21] C:\msys64\clang64\bin\cc.exe --target=x86_64-windows-gnu -DZYDIS_SHOULD_EXPORT -D_CRT_SECURE_NO_WARNINGS -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/include -IC:/msys64/mnt/vcpkg/buildtrees/zydis/x64-mingw-dynamic-dbg -IC:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src -isystem C:/msys64/mnt/vcpkg/installed/x64-mingw-dynamic/include -g -std=gnu11 -MD -MT CMakeFiles/Zydis.dir/src/SharedData.c.obj -MF CMakeFiles\Zydis.dir\src\SharedData.c.obj.d -o CMakeFiles/Zydis.dir/src/SharedData.c.obj -c C:/msys64/mnt/vcpkg/buildtrees/zydis/src/v4.1.0-1fdb2423e4.clean/src/SharedData.c
ninja: build stopped: subcommand failed.
```
</details>

--- a/ports/zydis/portfile.cmake
+++ b/ports/zydis/portfile.cmake
@@ -3,7 +3,7 @@ vcpkg_from_github(
     OUT_SOURCE_PATH SOURCE_PATH
     REPO zyantific/zydis
     REF "v${VERSION}"
-    SHA512 e07add4d43768ded02a238911fde6e74d2391abf8df282f774fca1a8c3fca3e97b03e90e0f3c7c0f3c75485fb29c0be4071d5e5b2e23dd5b8b1a864e3b713fbc
+    SHA512 177e84fedb3449e29ffb6c0b02a92066ba1aa8fb624facad5593902b8e08cb8ae0b20ff38c16987989c8e414d7484d09dab7917c00a8fe54aa9bab4bc90e275d
     HEAD_REF master
     PATCHES
         zycore.patch

--- a/ports/zydis/vcpkg.json
+++ b/ports/zydis/vcpkg.json
@@ -1,6 +1,6 @@
 {
   "name": "zydis",
-  "version-semver": "4.1.0",
+  "version-semver": "4.1.1",
   "description": "Fast and lightweight x86/x86-64 disassembler library.",
   "homepage": "https://zydis.re",
   "license": "MIT",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -10173,7 +10173,7 @@
       "port-version": 0
     },
     "zydis": {
-      "baseline": "4.1.0",
+      "baseline": "4.1.1",
       "port-version": 0
     },
     "zyre": {

--- a/versions/z-/zydis.json
+++ b/versions/z-/zydis.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "45ddfb739de0637d4c046d26dd91b88e6caef94e",
+      "version-semver": "4.1.1",
+      "port-version": 0
+    },
+    {
       "git-tree": "a08c5c766c41651280cb783e5dd0a8a5764f9700",
       "version-semver": "4.1.0",
       "port-version": 0

